users/router.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const passport = require('passport');
+
+const {User} = require('./models');
+
+const router = express.Router();
+
+const jsonParser = bodyParser.json();
+
+// Post to register a new user
+router.post('/', jsonParser, (req, res) => {
+  const requiredFields = ['username', 'password'];
+  const missingField = requiredFields.find(field => !(field in req.body));
+
+  if (missingField) {
```

Comment:
Yeah, I almost did this here but stuck with what we were using already.  Massive :+1: on that for me - doing this manually is asking for trouble.